integration/jnifuse/jnifuse-native/pom.xml

```diff
@@ -0,0 +1,68 @@
+<!--
+
+    The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+    (the "License"). You may not use this work except in compliance with the License, which is
+    available at www.apache.org/licenses/LICENSE-2.0
+
+    This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+    either express or implied, as more fully set forth in the License.
+
+    See the NOTICE file distributed with this work for information regarding copyright ownership.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <artifactId>alluxio-integration-jnifuse</artifactId>
+    <groupId>org.alluxio</groupId>
+    <version>2.5.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>alluxio-integration-jnifuse-native</artifactId>
+
+  <properties>
+    <!-- The following paths need to be defined here as well as in the parent pom so that mvn can -->
+    <!-- run properly from sub-project directories -->
+    <build.path>${project.parent.parent.parent.basedir}/build</build.path>
+  </properties>
+
+  <profiles>
+    <profile>
+      <id>fuseNative</id>
+      <activation>
+        <activeByDefault>false</activeByDefault>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-antrun-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>native_fuse_compile</id>
+                <phase>process-resources</phase>
+                <goals>
+                  <goal>run</goal>
+                </goals>
+                <configuration>
+                  <!--TODO:(maobaolong) make it can build for multi target-->
+                  <target>
+                    <mkdir dir="${project.build.directory}/native"/>
+                    <exec executable="make" failonerror="true" dir="${project.build.directory}/native">
+                      <arg line="-f ${project.basedir}/src/main/native/libjnifuse/Makefile"/>
+                    </exec>
+                    <copy todir="${project.build.directory}/classes">
```

Comment:
if the dynamic link libs are already in `alluxio-integration-jnifuse-native`, we can remove these two files in `lib/` and remove the library path set in https://github.com/Alluxio/alluxio/pull/12857 ?

Reply:
Sure